### DISCUSSION TITLE
Distributed Installation / Elastic Search

### DISCUSSION THREAD

**Discussion by Clankcoll** [telekom-security/tpotce#1079] [Q&A] [answered]
Hello i am Using the Distrbuted Installation with the Hive Master and the Hive Sensors, which is working great.

But i would like to know if there is an easy way to filter or to Create a Dashboard for the Single Sensors so that i can see Exactly and easy
what and who got on my German IP, Swiss IP, Kroatian IP etc.

Would be great if someone has an advice for me.

Kind regards 
C

**Comment by t3chn0m4g3** [comment, marked as answer]
Yes, you can filter for a couple of `t-pot*` fields you can use to accomplish this goal.

**Reply by Clankcoll** [reply]
Thanks seems to work well with the t-pot_hostname.keyword syn

Thank you for the fast reply
Also thanks for your Work and dedication to this wonderful Project (the Swiss Army Knif in the Swiss Tank)
